app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java

```diff
@@ -179,6 +180,26 @@ void openPdf() {
         }
     }
 
+    /**
```

Comment:
@ShrreyaBehll Why have you created this new function here? This is what is causing build failure.
We already have this function & you are copying it again.